pkg/golinters/misspell.go

```diff
@@ -153,3 +159,24 @@ func runMisspellOnFile(lintCtx *linter.Context, filename string, replacer *missp
 
 	return res, nil
 }
+
+func appendExtraWords(replacer *misspell.Replacer, extraWords []config.MisspellExtraWords) error {
+	var extra []string
```

Comment:
preallocation?

Reply:
preallocation is globally useless unless you have performance issues.